t/lib/warnings/op

```diff
@@ -1019,6 +1018,20 @@ sub fred () { 1 }
 EXPECT
 Constant subroutine main::fred redefined at - line 3.
 ########
+# op.c github #20742
+use constant fred => 1, 2;
+use constant fred => 2, 3;
+EXPECT
+OPTIONS regex
+Constant subroutine main::fred redefined at .*lib/constant\.pm line \d+
```

Comment:
Will that still be a `/` on non-UNIX platforms (e.g. MSWin32)?

Reply:
CI passed, including Win32, also the existing warning for non array constants:
```
C:\Users\Tony>perl -Mconstant=x,1 -Mconstant=x,2 -e0
Constant subroutine main::x redefined at c:/strawberry-perl-5.32.1.1-64bit-portable/perl/lib/constant.pm line 171.
```

Reply:
I suspect if it loaded as an absolute path (who does that?) It might not look like that. I doubt it matters.

Reply:
Can we resolve this and apply it?

Reply:
> I suspect if it loaded as an absolute path (who does that?) It might not look like that. I doubt it matters.

Normal use (via `use constant ...`) always has forward slashes, and the test does that.

Loading constant via an absolute path means `constant.pm` doesn't get an entry in %INC, so use constant is reloaded (and warns):
```
C:\Users\Tony>perl -WE "BEGIN { require 'c:\strawberry-perl-5.32.1.1-64bit-portable\perl\lib\constant.pm'} use constant x => 1; use constant x => 2;"
Subroutine import redefined at c:/strawberry-perl-5.32.1.1-64bit-portable/perl/lib/constant.pm line 55.
Constant subroutine main::x redefined at c:/strawberry-perl-5.32.1.1-64bit-portable/perl/lib/constant.pm line 171.
```
For a normal load when the entry in `@INC` uses native paths, the separator between the end of the base path and the "constant.pm" is still `/`:
```
C:\Users\Tony>perl -WE "BEGIN { tr[/][\\] for @INC } use constant x => 1; use constant x => 2"
Constant subroutine main::x redefined at c:\strawberry-perl-5.32.1.1-64bit-portable\perl\lib/constant.pm line 171.
```